sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_arg.asciidoc

```diff
@@ -386,6 +511,271 @@ constant property.
 
 |===
 
+The following table describes the overloaded operators for types `annotated_arg<T, ...>`
```

Comment:
@gmlueck I have added some more overloaded operators for logical and bit shift operations. I thought that the spec doc was getting very repetitive so I described the overloaded operators in a generic way before the table. Although, I am wondering whether the table itself is repetitive. We have listed all the overloaded operators in the class definition above so maybe here we can just have the description or have a generic template such as this: 

```c++
template <typename T2, typename PropertyList2,
          typename R = decltype(std::declval<T>() OP std::declval<T2>())>
R operatorOP(const annotated_arg<T2, PropertyList2> &other) const;
```

Reply:
I like this idea and think it would make the spec more readable.